packages/scss/src/components/_button.scss

```diff
@@ -1,92 +1,90 @@
 // MIXINS
 // ▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒▒
 
-@mixin buttonColoring($palette) {
-	background-color: _color($palette);
-	color: _color($palette, "text");
+@mixin buttonColoring($palette, $suffix: '') {
```

Comment:
Do we really need to add !important here ? Specifity alone should be sufficent to override those property. The risk is that it could snowball quickly and increase overall complexity.

Reply:
I tried to do without, and the class combinations led me to have to code this.